libs/ui-lib/lib/ocm/components/clusterConfiguration/ExternalPlatformDropdown.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup } from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { useField } from 'formik';
+import {
+  NUTANIX_CONFIG_LINK,
+  PlatformType,
+  VSPHERE_CONFIG_LINK,
+  getFieldId,
+} from '../../../common';
+
+const INPUT_NAME = 'platform';
+const fieldId = getFieldId(INPUT_NAME, 'input');
+export type ExternalPlatformType = Extract<PlatformType, 'none' | 'nutanix' | 'oci' | 'vsphere'>;
+
+type ExternalPlatformDropdownProps = {
+  showOciOption: boolean;
+  selectedPlatform: ExternalPlatformType;
+  disabledOciTooltipContent: React.ReactNode;
+  isOciDisabled: boolean;
+  onChange: (selectedPlatform: ExternalPlatformType) => void;
+  dropdownIsDisabled: boolean;
+};
+
+export type ExternalPlatformInfo = {
+  label: string;
+  href: string;
+  tooltip: string;
+};
+
+export const externalPlatformTypes: Record<ExternalPlatformType, ExternalPlatformInfo> = {
+  none: {
+    label: 'None',
+    href: '',
+    tooltip: '',
+  },
+  nutanix: {
+    label: 'Nutanix',
+    href: NUTANIX_CONFIG_LINK,
+    tooltip: '',
+  },
+  oci: {
+    label: 'Oracle  (Requires a custom manifest)',
+    href: 'www.google.es',
+    tooltip:
+      "To integrate with an external partner (for example, Oracle Cloud), you'll need to provide a custom manifest.",
+  },
+  vsphere: {
+    label: 'vSphere',
+    href: VSPHERE_CONFIG_LINK,
+    tooltip: '',
+  },
+};
+
+export const ExternalPlatformDropdown = ({
```

Comment:
Not sure what the problem is, but there is a console error regarding a missing `key` prop which seems to come from this component.